File: tyler-stamm/create-phone-number.js
function createPhoneNumber(numbers){
  // return ten digits the same, but in format of ""(000) 000-0000"
  // return as string
  // create new variable for format of numbers
  // use for loop to loop[ through it]
  // use .replace to replace x with the array
  // use .toString to convert to string
  var format = "(xxx) xxx-xxxx"
  for( let i = 0; i <numbers.length; i++) {
    format = format.replace("x", numbers[i])  

  }
  return format.toString();
}

File: tyler-stamm/create-phone-number.py
def create_phone_number(n):
    # set format to (xxx) xxx-xxxx
    # loop through the string
    # use .replace to replace x with that specific iteration
    format = "(xxx) xxx-xxxx"
    for num in n:
        format = format.replace("x", str(num), 1)
    return format
